chart/templates/configmaps/extra-configmaps.yaml

```diff
@@ -39,6 +39,9 @@ metadata:
     "helm.sh/hook": "pre-install,pre-upgrade"
     "helm.sh/hook-delete-policy": "before-hook-creation"
     "helm.sh/hook-weight": "0"
+    {{- if $configMapContent.annotations }}
+      {{- toYaml $configMapContent.annotations | nindent 4 }}
+    {{- end }}
```

Comment:
```suggestion
    {{- with $configMapContent.annotations }}
      {{- toYaml . | nindent 4 }}
    {{- end }}
```

Reply:
I looked this up. The changes in line with the recent changes to these specific files. Code readability took precedence as I didnt want to have 2 different code patterns in the same file when it has no affect on intended functionality. Thanks for suggesting, and I will be sure to keep this in mind in my future PRs as well